gitops/platform/teams/backend/namespaces-staging-values.yaml

```diff
@@ -66,10 +130,48 @@ namespaces:
           - operator : In
             scopeName: PriorityClass
             values: ["high"]
+    networkPolicies:
+    - name: allow-egress-same-namespace
+      spec:
+        podSelector: {}
+        policyTypes:
+        - Egress
+        egress:
+        - to:
+          - podSelector: {}
+    - name: allow-ingress-same-namespace-and-from-ui
+      spec:
+        podSelector: {}
+        policyTypes:
+        - Ingress
+        ingress:
+        - from:
+          - podSelector: {}
+          - namespaceSelector:
+              matchLabels:
+                kubernetes.io/metadata.name: ui
+    roleBindings:
+    - name: view
+      roleRef:
+        apiGroup: rbac.authorization.k8s.io
+        kind: ClusterRole
+        name: view
+      subjects:
+      - apiGroup: rbac.authorization.k8s.io
+        kind: Group
+        name: backend-team-view
+
   checkout:
     labels:
       additionalLabels:
         app.kubernetes.io/created-by: eks-workshop
+      podSecurityAdmission:
```

Comment:
With kyverno I unforced restricted PSS wonder if this will clash

Reply:
@allamand  If you are setting the annotation via kyverno mutation on update/create then we need to remove from GitOps, because GitOps will revert. Remember the source of truth should be in git. 
We can add ignoreDifference in the argocd app so it doesn't revert the change, and the scenario is that helm chart would set it to baseline, and that when using kyverno it will be set by policy regardless of gitops config.